README.md

```diff
@@ -1,2 +1,42 @@
-# user-provisioning-scripts-js
-Command line tools used to demonstrate how to provision users, add them to a group and create a webrtc phone for them in Genesys Cloud
+# Introduction
+
+Welcome to the user-provisioning-scripts-js repo. This repo contains a node.js-based command line script that will take a comma-separated
+
+1. Create a user in the Genesys Cloud platform.
+2. Assign the user to a group within Genesys Cloud.
+3. Assign the user to a specific role within Genesys Cloud.
+4. Create a WebRTC phone for the user and then assign the newly created phone as the default station for the user.
+
+## Installing and running the code
+
+All code was run using node version 14.3.0. The code should work with Node 11 and above. To run the code you need to first:
+
+1. Ensure that node 11 are higher is running on your server.
+2. Run npm i to install all of the pacakges and dependencies.
+3. Setup a OAuth Client Credential grant in your OAUTH instance. For purposes of this tutorial, the OAuth Client was configured with the Communicate Admin and Telephony Admin roles.
+
+4. Set the following environment variables:
+
+   - GENESYS_CLIENT_ID=<<YOUR CLIENT ID>>
+   - GENESYS_CLIENT_SECRET=<<YOUR CLIENT SECRET>>
+
+   **Note**: You can also set the above environment variables in .env file.
+
+5. To run the test cases, you can run: npm run test
+6. To run the actual code, you can change to the src directory and run: node index.js ../data/userdata.csv
+
+   **Note**: This code will create the users against your organization, assign them to a group and a role so please be aware of this and be careful. This code is a tutorial for how to provision users. It has mininmal error handling and no rollback logic if something goes wrong.
+
+## Code Organization
+
+    data/                   <= Contains a single csv file mapping out the user fields to map
```

Comment:
This whole section under Code Organization doesn't display as its written; rendering plain text in HTML condenses all contiguous whitespace to a single space. I'd recommend putting this in a table. If you'd prefer this look, put it in a code block without specifying a language to render preformatted text without syntax highlighting.

Reply:
VSCode lied to me.  I was using the renderer and it displayed properly.

Reply:
Standard markdown syntax is to have an implied code block if text is indented. I disabled that syntax for the site's markdown parsers because the overwhelming majority of the formatting matches weren't intended, usually just collisions with other syntax.